### PR TITLE
fix(sticky): move to different sticky library

### PR DESCRIPTION
**Pull request by techninja**
This is fully tested with both storybook, mobile, and frontend build! But... it fails creating new snapshots. Needs a little kicking, and likely another set of eyes.

### CODE REVIEW

**Comment by evanmwillhite** [comment]
## Frontend opinion alert.:rotating_light: 

**tl;dr**: I'd rather see if we can stay with a library that uses [position: sticky](https://caniuse.com/#feat=css-sticky) in CSS as it provides the smoothest experience, even if IE11 users just don't benefit from it (in which case we can ignore that other library altogether and just use CSS). 

I leave the IE11 decision up to the client, but wanted to speak out against this before we went too far down this other road. These old methods use fixed positioning, which does 1 good thing (keeps it on screen) and a lot of other terrible things like taking the element out of the context of the layout completely. So then a lot of JS work has to be done to recalculate where the heck the element was in the first place and place it there using pixels values (which also have to be recalculated if the window is resized, yada yada). Yuck.

`/endFrontendOpinionAlert`:rotating_light:

**Comment by brandonhundt** [comment]
Recapping a Slack discussion:

I'm alright if it isn't sticky for IE11 users because its ultimately something that wouldn't break the experience.

**Comment by evanmwillhite** [comment]
Closing in favor of https://github.com/PublicRadioInternational/pri-component-library/pull/55. Can reopen later if necessary.
